src/core/l10n.js

```diff
@@ -17,7 +17,7 @@ const base = {
   editors_note: "Editor's note",
   editors: "Editors:",
   example: "Example",
-  fig: "Fig. ",
+  fig: "Figure ",
```

Comment:
As my screenreader also doesn't know how to expand the "fig." abbr, I'm inclined to support this change.